builder/parser/testfiles-negative/html-page-yes-really-thanks-lk4d4/Dockerfile

```diff
@@ -1,2 +0,0 @@
-<html>
```

Comment:
This wasn't a failure test due to the invalid command, it was failing because it didn't have any args - which this PR will now allow.  So from a "parser" perspective this test is ok and therefore should be removed as a "negative" test.  Instead, we'll catch the invalid command part of this test later on in the builder, not the parser - and we already have "invalid command" tests in the intergration-cli test suite.